framework/sysdeps/unix/child_debug.cpp

```diff
@@ -187,8 +187,39 @@ print("..Done..")
 end
 )gdb";
 
-static void child_crash_handler(int, siginfo_t *si, void *ucontext)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-security"
+template <typename... Args> [[noreturn]] static void
+terminate_with_ignored_crash(const char *msg, Args... params)
 {
+    // we can't really recover from a signal, so we terminate the
+    // process in a way the parent can tell it was an ignored error
+    log_message_skip(-1, IgnoredMceCategory, msg, params...);
+    log_warning("MCE was delivered to or is related to this thread");
```

Comment:
Note, it's a horrible idea to use the logging framework from inside a SIGBUS handler. We may deadlock.  Without it, I'd need to find some other way to communicate what happened.